server/embed/config.go

```diff
@@ -56,19 +56,22 @@ const (
 	ClusterStateFlagNew      = "new"
 	ClusterStateFlagExisting = "existing"
 
-	DefaultName                        = "default"
-	DefaultMaxSnapshots                = 5
-	DefaultMaxWALs                     = 5
-	DefaultMaxTxnOps                   = uint(128)
-	DefaultWarningApplyDuration        = 100 * time.Millisecond
-	DefaultWarningUnaryRequestDuration = 300 * time.Millisecond
-	DefaultMaxRequestBytes             = 1.5 * 1024 * 1024
-	DefaultMaxConcurrentStreams        = math.MaxUint32
-	DefaultGRPCKeepAliveMinTime        = 5 * time.Second
-	DefaultGRPCKeepAliveInterval       = 2 * time.Hour
-	DefaultGRPCKeepAliveTimeout        = 20 * time.Second
-	DefaultDowngradeCheckTime          = 5 * time.Second
-	DefaultAutoCompactionMode          = "periodic"
+	DefaultName                             = "default"
+	DefaultMaxSnapshots                     = 5
+	DefaultMaxWALs                          = 5
+	DefaultMaxTxnOps                        = uint(128)
+	DefaultWarningApplyDuration             = 100 * time.Millisecond
+	DefaultWarningUnaryRequestDuration      = 300 * time.Millisecond
+	DefaultMaxRequestBytes                  = 1.5 * 1024 * 1024
+	DefaultMaxConcurrentStreams             = math.MaxUint32
+	DefaultGRPCKeepAliveMinTime             = 5 * time.Second
+	DefaultGRPCKeepAliveInterval            = 2 * time.Hour
+	DefaultGRPCKeepAliveTimeout             = 20 * time.Second
+	DefaultDowngradeCheckTime               = 5 * time.Second
+	DefaultAutoCompactionMode               = "periodic"
+	DefaultAuthToken                        = "simple"
+	DefaultExperimentalCompactHashCheckTime = time.Minute
+	DefaultWaitClusterReadyTimeout          = 5 * time.Second
```

Comment:
We already reverted the waitClusterReady feature in https://github.com/etcd-io/etcd/pull/16677, why do we still keep this default value?

Reply:
Ups, rebase error.

Reply:
Fixed.